go/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.8
+
+WORKDIR /go/src/github.com/Originate/exocom/go/
+COPY . .
+
+WORKDIR exocom
+RUN go install
+RUN rm -rf /go/src/
+
+CMD /go/bin/exocom
```

Comment:
That should work. I'm just not sure what version we need for the docker image and this way it build it on the machine it runs.